### PR TITLE
Support generate_load_scripts.

### PR DESCRIPTION
**Pull request by isaacabraham**

### CODE REVIEW

File: src/paket.fs [hunk 1]
```diff
@@ -246,13 +246,14 @@ let private createDependenciesProvider () =
...
                              "references: strict"; "framework:"; "content: none"; "copy_content_to_output_dir: always"
-                             "import_targets:"; "copy_local:"; "redirects:"; "strategy:"; "lowest_matching:" ]
+                             "import_targets:"; "copy_local:"; "redirects:"; "strategy:"; "lowest_matching:"; "generate_load_scripts" ]
                             |> concatAndLift
                         | PaketTag "nuget" -> send word
                         | PaketTag "source" -> [ "https://api.nuget.org/v3/index.json"; "https://nuget.org/api/v2" ] |> concatAndLift
                         | PaketTag "framework:" -> [ "net35"; "net40"; "net45"; "net46"; "auto-detect" ] |> concatAndLift
                         | PaketTag "redirects:" -> [ "on"; "off"; "force" ] |> concatAndLift
                         | PaketTag "strategy:" -> [ "min"; "max" ] |> concatAndLift
+                        | PaketTag "generate_load_scripts:" -> [ "true"; "false" ] |> concatAndLift
```
Comment (forki) [inline src/paket.fs:256]: I think it's on vs. off

**Review by forki — CHANGES_REQUESTED** [review]

File: src/paket.fs (hunk 1 above)
Comment (forki) [reply src/paket.fs:256]: or maybe it isn't and I just did it wrong in lockfile

Comment (forki) [reply src/paket.fs:256]: ok just checked. all is well.

**Review by forki — APPROVED** [review]

**Comment by Krzysztof-Cieslak** [comment]
Uhh... not sure how I've missed this PR :(

**Comment by isaacabraham** [comment]
Eh? This has been in Ionide for ages now, hasn't it?

**Comment by Krzysztof-Cieslak** [comment]
This PR adds `generate_load_scripts` to `Paket.dependencies` autocomplete

**Pull request merged**